.gitignore

```diff
@@ -1,5 +1,7 @@
-KituraCache.xcodeproj
+*.xcodeproj/*
 Packages
```

Comment:
We should remove Packages

Reply:
Keeping it in would prevent accidental commits of packages that are being edited with `swift package edit`

Reply:
Fair enough, I'll close this request